datasets/ky/judicial/crawler.py

```diff
@@ -73,10 +76,17 @@ def crawl_judges(context: Context, url, default_position, min, max):
 
 
 def crawl(context: Context):
-    crawl_judges(context, context.data_url, "Chief Justice", 1, 1)
-
-    appeal_court_url = urljoin(context.data_url, "court-of-appeal")
-    crawl_judges(context, appeal_court_url, "Justice of the Court of Appeal", 3, 10)
-
-    grand_court_url = urljoin(context.data_url, "grand-court")
-    crawl_judges(context, grand_court_url, "Judge of the Grand Court", 3, 10)
+    doc = context.fetch_html(context.data_url, cache_days=1)
+    profile_links = [
+        link
+        for link in doc.xpath('//ul[@id="menu-judicial-officers"]//a/@href')
+        if link != "#"
+    ]
+    assert len(profile_links) >= 6, profile_links
+    for url in profile_links:
+        doc = context.fetch_html(url, cache_days=1)
+        profile_links = doc.xpath(
```

Comment:
(nit) let's use a different variable name, since you're currently iterating over it. It works, since you're not actually changing profile_links (then it would explode I think) but just assigning a new value to it, but it's an extra thing to think about. Maybe `person_urls` is better?